src/core/src/sys/system.rs

```diff
@@ -38,6 +38,13 @@ pub struct System {
     external_temperature: Option<f64>,
 }
 
+
+/// Struct that can be used to query the
+/// local interactions. As opposed to the `System`,
+/// it is `Sync`.
+#[derive(Copy, Clone)]
+pub struct LocalPotentials<'a>(&'a Configuration, &'a LocalInteractions);
```

Comment:
I don't really like the tuple struct here, partly because accessing the fields through `self.0` and `self.1` make the code harder to read. Also, we might not need to expose this in the public API. See my main comment for an alternative proposal.